content/post/18-the-user-interface-of-the-violin/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The User Interface of the Violin"
+date: 2014-08-07T00:00:00-07:00
+draft: false
+categories:
+- zen
+- general
+- software
+
+---
+Earlier today, I met a friend for breakfast who’s an outstanding professional UX designer.  I was curious about something:
+
+“How do you balance the need to give people a user interface they’re familiar with and can do something with right away, against the opportunity to innovate and do new things that may take more time to learn?”
+
+He gave a beautiful analogy in response.
+
+{{< imgproc "images/violin.png" Fit "400x500" >}}Violin{{</imgproc>}}
```

Comment:
There's no need to explicitly label this a violin with a caption. People will understand what it is when they see it. :)

```suggestion
{{< imgproc "images/violin.png" Fit "400x500" />}}
```